lib/expand/bootstrap.go

```diff
@@ -200,3 +205,17 @@ func getPeerAddrAndToken(ctx operationContext, role string) (peerAddr, token str
 	}
 	return peerAddr, instructions.Token, nil
 }
+
+func patchSELinuxConfig(ctx operationContext) error {
```

Comment:
Configuring custom values is tricky on joining nodes - in this case, in order to set up the custom `vxlan` port, the installer needs to bootstrap the policy with enough rules to make it to dial the cluster and, after learning about the overrides - update the port configuration with the new value for the port.